### PR TITLE
Displays sample counts in prep/sample template.

### LINKED ISSUE #608: show number of samples
The study/prep/sample summaries should show the number of samples.

### PR DESCRIPTION
**Pull request by JWDebelius**
Fixes #608

### CODE REVIEW

File: qiita_pet/templates/metadata_summary.html [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 <h1>Metadata Summary for Study {{study_title}}</h1>
 
+<div>
+    <p>There are <strong>{{num_samples}}</strong> samples in this study.</p>
```
Comment (wasade) [inline qiita_pet/templates/metadata_summary.html]: There is an extra space, only need once space on the side of `num_samples`

**Comment by wasade** [comment]
One comment, :+1: otherwise

File: qiita_pet/templates/metadata_summary.html (hunk 1 above)
Comment (wasade) [inline qiita_pet/templates/metadata_summary.html]: ya, im an idiot

Comment (JWDebelius) [inline qiita_pet/templates/metadata_summary.html]: I think it needs both spaces to render sanely.

**Pull request merged**